apps/web-giddh/src/app/gst/gstR2/gstR2.component.html

```diff
@@ -1,6 +1,6 @@
-<div class="help-support-aside setting-sidebar-wrapper" *ngIf="asideGstSidebarMenuState === 'in'">
+<div class="help-support-aside setting-sidebar-wrapper gst-sidebar-wrapper" *ngIf="asideGstSidebarMenuState === 'in'" (clickOutside)="isMobileScreen ? asideGstSidebarMenuState = 'out' : ''" [exclude]="'#primary-new-header'">
     <gstr-sidebar (navigateEvent)="handleNavigation($event)" [activeCompanyGstNumber]="activeCompanyGstNumber"
-        [selectedGstModule]="selectedGstr" class="gst-sidebar"></gstr-sidebar>
+        [selectedGstModule]="selectedGstr" [commonLocaleData]="commonLocaleData" [localeData]="localeData" class="gst-sidebar"></gstr-sidebar>
```

Comment:
remove input of localeData and commonLocaleData from this component because this component is not getting used.

Reply:
reverted